content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -418,6 +418,12 @@ The access modes are:
 * ReadWriteOncePod -- the volume can be mounted as read-write by a single Pod.
   This is only supported for CSI volumes and Kubernetes version 1.22+.
 
+{{< note >}}
+ReadWriteOnce access mode still can allow multiple pods to access the volume when the pods are running on the same node.
+Use ReadWriteOncePod access mode if you want to ensure that only one pod across whole cluster can read that PVC or write to it. 
+For more information please check this [blog](https://kubernetes.io/blog/2021/09/13/read-write-once-pod-access-mode-alpha/)
```

Comment:
```suggestion
The blog article [Introducing Single Pod Access Mode for PersistentVolumes](/blog/2021/09/13/read-write-once-pod-access-mode-alpha/) covers this in more detail.
```

Because you need to enable an alpha feature to use it, I don't think a `{{< note >}}` callout is justified.

We could instead expand
> ```markdown
> * ReadWriteOnce -- the volume can be mounted as read-write by a single node
> ```

to make it clear about the implications, whatever feature gates your cluster is running with.

Reply:
If rewriting lines 415 to 419, consider using a [definition list](https://www.markdownguide.org/extended-syntax/#definition-lists) rather than the current unordered list.

Reply:
Hi @sftim , i have addressed the comments as you suggested. PTAL!